scripts/tools/eslint-plugin-prettier-internal-rules/no-single-doc-concat.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
```

Comment:
It would be good to add a comment about removing `concat` and this rule?
```suggestion
// TODO: Remove this rule when deprecated `concat` is removed.
```